docs/docs/configuration/mapper.mdx

```diff
@@ -68,6 +68,37 @@ public partial class CarMapper
 }
 ```
 
+#### IgnoreAttribute
```

Comment:
The documentation of Mapperly usually uses describing titles instead of the names of the specific API. Probably `Ignore a member at definition`